src/tools/interop/interop.cpp

```diff
@@ -119,8 +119,8 @@ uint32_t CurrentThreadCount;
 
 uint16_t CustomPort = 0;
 
-const char* UrlPath = "/";
 bool CustomUrlPath = false;
+std::vector<const char*> Urls;
```

Comment:
If you're using `vector`, maybe worth while to use `string` too?

Reply:
What benefit would there be, though?

Reply:
you don't have to manage the memory?